### PR TITLE
RATIS-1331. Make GrpcTLS config truststore param list.

### PR DESCRIPTION
**Pull request by bharatviswa504**
## What changes were proposed in this pull request?

This Jira is to make truststore param a list of trusted CA certificate.
This will be needed for SCM HA.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/RATIS-1331

### CODE REVIEW

**Comment by bharatviswa504** [comment]
cc @xiaoyuyao / @bshashikant for review.

**Review by mukul1987 — APPROVED** [review]
+1, LGTM

File: ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcTlsConfig.java [hunk 1]
```diff
@@ -75,14 +77,23 @@ public boolean getMtlsEnabled() {
...
     this.trustStore = trustStore;
     this.mTlsEnabled = mTlsEnabled;
     this.fileBasedConfig = false;
   }
 
+  public GrpcTlsConfig(PrivateKey privateKey, X509Certificate certChain,
+      X509Certificate trustStore, boolean mTlsEnabled) {
+    this.privateKey = privateKey;
+    this.certChain = certChain;
+    this.trustStore = Arrays.asList(trustStore);
```
Comment (adoroszlai) [inline ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcTlsConfig.java:92]: Suggestion: use `Collections.singletonList` instead of `Arrays.asList`.

Comment (bharatviswa504) [reply ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcTlsConfig.java]: Done

**Pull request merged**